doc-content/enterprise-only/getting-started-kogito/con-migrate-dmn-to-kogito-overview.adoc

```diff
@@ -0,0 +1,63 @@
+[id="con-migrate-dmn-to-kogito-overview_{context}"]
+= Migration of a DMN service to {KOGITO} microservice
+
+[role="_abstract"]
+You can migrate decision services that are implemented using DMN models to {KOGITO} microservices by moving the DMN resources from KJAR artifacts to the respective {KOGITO} archetype.
+
+
+[id="ref-migrate-dmn-to-kogito-considerations_{context}"]
+== Major changes and migration considerations
+
+The following table describes the major changes and features that affect migration from KIE v7 API and KJAR to {KOGITO} deployments:
+
+.Migration to DMN considerations
+[cols="30%,35%,35%" options="header"]
+|===
+|Feature
+|In {KIE_SERVER} API
+|In {KOGITO} artifact
+
+|DMN Models
+|stored in `src/main/resources` of KJAR
+|copy as-is to `src/main/resources`
+
+|Object models (POJOs) required for {KIE_SERVER} generic marshalling
+|managed using Data Model object editor in {CENTRAL}
+|object model editing is not supported
+
+|`DMNRuntimeListener`
+|configured using a system property or `kmodule.xml`
+|must be configured using CDI, by annotating the `DMNRuntimeEventListener` with CDI’s `@ApplicationScope` annotation
+
+|Other configuration options
+|configured using system property or `kmodule.xml`
+|except `DMNRuntimeEventListener`, only default values are considered and no override of configuration is supported
+
+|{KIE_SERVER} Client API
+|used in conjunction with object models to interact with a KJAR that is deployed on {KIE_SERVER}
+|as for object models, this feature is not supported
+
+|REST API
+|when KJAR is deployed on {KIE_SERVER}, the applications that interact with an endpoint specific to a DMN model, uses the same API on {KOGITO} deployment
+|advanced support for specific DMN model generation. For more information, see {URL_DEVELOPING_DECISION_SERVICES}#dmn-execution-con_dmn-models[_DMN model execution_].
+
+|Test scenarios
+|run with JUnit activator
+|analogous JUnit activator is available on {KOGITO}
+
+|===
+
+NOTE: The features that are not mentioned in the previous table are either not supported or not required in a cloud-native {KOGITO} deployment.
+
+[id="con-migration-dmn-strategy_{context}"]
+== Migration strategy
+
+[role="_abstract"]
+When migrating a DMN project to {KOGITO} project, you must first migrate external applications that are interacting with decision services on the {KIE_SERVER} using REST endpoints that are specific to DMN models. For more information about REST endpoints for specific DMN models, see {URL_DEVELOPING_DECISION_SERVICES}#ref-rest-api-dmn-endpoints_dmn-models[_REST endpoints for specific DMN models_].
+
```

Comment:
@tarilabs as per the IBM style guide, we need to avoid the usage of `should`, and `should` must be replaced with `must`. Also to make the paragraph easy to understand, rewriting the paragraph as follows:

"When migrating a DMN project to {KOGITO} project, you must first migrate external applications that are interacting with decision services on the {KIE_SERVER}. You can use the REST endpoints that are specific to DMN models. After using the REST endpoints, it is easier to migrate the external applications from {KIE_SERVER} to {KOGITO} deployment. For more information about REST endpoints for specific DMN models, see {URL_DEVELOPING_DECISION_SERVICES}#ref-rest-api-dmn-endpoints_dmn-models[_REST endpoints for specific DMN models_]."